### PR TITLE
fix: Throw exception on invalid color instead of falling back to Red

### LINKED ISSUE #11229: An invalid color is returned as a red color
### Current behavior

An invalid color, e.g. something that can't be parsed, is returned as a red color for `Colors.FromArgb`

https://github.com/unoplatform/uno/blob/4092e2f08d6cbf385631a5a89c68f7036e7d4fcf/src/Uno.UWP/UI/Colors.cs#L105

### Expected behavior

Ref: https://github.com/unoplatform/uno/pull/11110#pullrequestreview-1285604621

In general invalid values throws an exception.

OTOH from previous experience (silverlight/moonlight) I know that it's often easier to debug such problem visually as the exception is not always easy to map back to the invalid value.

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #11229

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWP/UI/Colors.cs [hunk 1]
```diff
@@ -250,10 +250,9 @@ public static Color FromARGB(string colorCode)
 			}
 			else
 			{
-				r = 0xFF;
-				g = 0x0;
-				b = 0x0;
+				throw new ArgumentException($"Cannot parse color '{colorCode}'.");
 			}
+
```
Comment (jeromelaban) [inline src/Uno.UWP/UI/Colors.cs:255]: @spouliot maybe that's the red we were looking for ? :)

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UWP/UI/Colors.cs (hunk 1 above)
Comment (spouliot) [reply src/Uno.UWP/UI/Colors.cs:255]: I'll check that out asap!

**Review by carldebilly — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

File: src/Uno.UWP/UI/Colors.cs (hunk 1 above)
Comment (spouliot) [reply src/Uno.UWP/UI/Colors.cs:255]: No, the pinkish issue does not hit that code path 😢 

And now I realize I opened the original issue... no wonder it ringed a bell (but not loud enough it seems).

**Pull request merged**